zetaclient/chains/sui/signer/signer.go

```diff
@@ -177,8 +179,24 @@ func (s *Signer) signTx(ctx context.Context, tx models.TxnMetaData, zetaHeight,
 		return "", errors.Wrap(err, "unable to get digest")
 	}
 
+	var (
+		// calculate artificial keysign height
```

Comment:
can we maybe precise a bit what is artifical ks height and why cantor pair is used?

Reply:
sure https://github.com/zeta-chain/node/pull/4487/commits/6a1f5ebac6fa9eff2bcc2e4423d5e84df9dd4dfd